node.js

```diff
@@ -237,6 +238,21 @@ module.exports = {
     }
   },
 
+  loadFeature: function loadFeature (usage, code) {
+    code = code.replace(/[^\w-]/g, '')
+    if (!usage[code]) {
+      // eslint-disable-next-line security/detect-non-literal-require
+      var compressed = require('caniuse-lite/data/features/' + code + '.js')
+      var usageData = feature(compressed).stats
+      usage[code] = { }
```

Comment:
`usage` cache was named for usage statistics. We need to rename it if we want to use it for feature cache.

Reply:
Not fully sure if I understand it, it's a local variable... Are you talking about `browserslist.usage`? Ok, make sense to use a differente cache. What's a better name, `browserslist.features` or `browserslist.supports`?

Reply:
Yeap, I do not like that name of local variabe `usage` because it doesn’t explain the content in the variable. Seems like it was copied from `loadCountry` when it was related to “usage statistics”.

Reply:
Done, I've renamed it to `features`.